README.md

```diff
@@ -2,21 +2,54 @@
 
 # JavaScript Objects Diagnostic
 
+## Prerequisites
+
+-   [JavaScript Objects Modeling](https://github.com/ga-wdi-boston/js-objects-modeling)
+-   [JavaScript Objects: Referencing Own Properties](https://github.com/ga-wdi-boston/js-objects-this)
+-   [JavaScript Constructors](https://github.com/ga-wdi-boston/js-objects-constructors)
+-   [JavaScript Prototypes](https://github.com/ga-wdi-boston/js-objects-prototypes)
+
 ## Instructions
 
 1.  Fork and clone this repository.
+1.  Change into the new directory.
 1.  Create a new branch, named `response`.
 1.  Checkout the `response` branch.
 1.  Install dependencies by running `npm install`
 1.  Follow the directions given in [lib/diagnostic.js](lib/diagnostic.js).
 1.  Check your responses by running `grunt test`.
 1.  Before the alotted time is up, push to your fork and issue a pull request.
 
+Your pull request description should contain a "fist to five" for comfort and
+clarity. Additionally, you should mention the resources you used to help you
+complete this diagnostic. For example:
+
+```md
+Comfort: 3
+Clarity: 3
+
+I used Google and my class notes to help with this diagnostic.
+```
+
 You may wish to refer to ["How do I submit diagnostics?"](https://github.com/ga-wdi-boston/meta/wiki/Diagnostics)
 and other [FAQs](https://github.com/ga-wdi-boston/meta/wiki/) related to
 [forking, cloning](https://github.com/ga-wdi-boston/meta/wiki/ForkAndClone),
 and [pull requests](https://github.com/ga-wdi-boston/meta/wiki/PullRequest).
 
+You may use **any resource** other than each other to complete this diagnostic.
+This includes referencing talk materials, appropriate documentation, and
+searching for help online.
+
+You should be running `grunt nag` before diagnosing any bugs, since it finds
+some of the most common sources of errors. After `grunt nag` passes, you should
+run `grunt test` to run the included tests. Tests will tell you whether of not
+your responses are correct.
+
+It's probably a good idea to include the above verbatim in each diagnostic.
```

Comment:
Good call on including this above section, but the snippet below in ln49 to52 are template instructions and can be removed.